Api/Models/Venue.cs

```diff
@@ -73,8 +81,16 @@ public EmbedBuilder ToEmbed(string uiUrl, string bannerUrl)
                                "Pacific Standard Time" => "PST",
                                "Atlantic Standard Time" => "AST",
                                _ => opening.Start.TimeZone
-                           }).Append(')');
+                           }).Append(" (ND/PMN)");
+                    if (opening.Start.NextDay)
+                    {
+                        stringBuilder.Append(" (");
+                        stringBuilder.Append(opening.Day.Next().ToShortName());
+                        stringBuilder.Append(")");
+                    }

```

Comment:
Had problem previously where the time on the venue was actually marked as "next day" but Veni's embeds don't show it. This resolves that.